aptos-move/aptos-aggregator/src/delta_change_set.rs

```diff
@@ -416,6 +416,10 @@ mod test {
                 Some("Error message from BadStorage.".to_string()),
             )))
         }
+
+        fn generate_aggregator_v2_id(&self) -> Self::IdentifierV2 {
```

Comment:
`delta_change_set` is more of an aggregator v1 file. Can we have this function in a aggregator v2 file?